ahnlich/ai/src/engine/ai/mod.rs

```diff
@@ -1,10 +1,7 @@
-mod models;
-use ahnlich_types::keyval::{StoreInput, StoreKey};
-use models::ModelInfo;
-use std::num::NonZeroUsize;
+pub mod models;
+use once_cell::sync::Lazy;
 
-pub trait AIModelManager {
-    fn embedding_size(&self) -> NonZeroUsize;
-    fn model_ndarray(&self, storeinput: &StoreInput) -> StoreKey;
-    fn model_info(&self) -> ModelInfo;
-}
+use ahnlich_types::ai::AIModel;
+
+pub(crate) static AHNLICH_AI_SUPPORTED_MODELS: Lazy<Vec<AIModel>> =
```

Comment:
Supported models are all the models inside AIModel enum... so rather than do this, look into strum to be able to iterate an enum